src/dotnet/commands/dotnet-restore/NuGet3.cs

```diff
@@ -10,13 +10,13 @@ namespace Microsoft.DotNet.Tools.Restore
 {
     internal static class NuGet3
     {
-        public static int Restore(IEnumerable<string> args, bool quiet)
+        public static int Restore(IEnumerable<string> args)
         {
             var prefixArgs = new List<string>();
-            if (quiet)
+            if (!args.Any(s => s.Equals("--verbosity", StringComparison.OrdinalIgnoreCase) || s.Equals("-v", StringComparison.OrdinalIgnoreCase)))
             {
                 prefixArgs.Add("--verbosity");
-                prefixArgs.Add("Error");
+                prefixArgs.Add("minimal");
```

Comment:
The default verbosity seems to be "Information", the level above "verbosity." Thus we need prefixArgs to set it to minimal. 

IMO (and the O of the folks in the issues about this) Information is too spammy. We could push the change all the way down, but I think it is very reasonable that the CLI, as a thing that uses NuGet as a dependency, gets to decide what the CLI shows CLI consomers.